domains/_github-pages-challenge-rubinskiy.bob.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Rubinskiy",
+        "discord": "rubinskiy#5258"
```

Comment:
![Screenshot_2023-08-07-10-17-22-80](https://github.com/is-a-dev/register/assets/43115551/e017795f-4a33-4dba-994c-29f3c6ac10c4)

There is a validation issue. U need to have a valid username and a email.